.circleci/config.yml

```diff
@@ -39,9 +48,35 @@ jobs:
           name: OWASP
           command: |
             ./gradlew dependencyCheckAnalyze --info
+  Scan Docker Image:
+    executor: anchore/anchore_engine
+    steps:
+      - setup_remote_docker:
+          docker_layer_caching: true
+      - checkout
+      - attach_workspace:
+          at: /tmp/workspace
```

Comment:
I've never used workspaces before. Why does the job "Full Build" persist to `workspace`, but the "Scan Docker Image" job attach to `/tmp/workspace`?